source/helper.h

```diff
@@ -66,6 +75,15 @@ struct Parameters {
   double surface_gravity;
   /* Number of grid points */
   size_t N;
+  
+  /* Arrays for variable abundance radiative losses */
+  double log10_loss_rate_array[101][7][7];
+  double log10_temperature_array[101];
+  double initial_density;
```

Comment:
Hardcoding the size of these arrays makes me nervous, especially since the size appears in three separate places in the code: here, the read function, and the radiative loss calculation. Could these be vectors instead so that their sizes don't have to be declared here?